lib/awful/hotkeys_popup/keys/tmux.lua

```diff
@@ -0,0 +1,77 @@
+---------------------------------------------------------------------------
+--- tmux hotkeys for awful.hotkeys_widget
+--
+-- @autor nahsi nashi@airmail.cc
```

Comment:
should be `author`

that's causing Travis build to fail

after fixing that please also squash all your commits